tests/unit/Foundation/ContainerUtilTest.php

```diff
@@ -17,10 +17,7 @@ class ContainerUtilTest extends TestCase
 {
     private $container;
 
-    /**
-     * @inheritDoc
-     */
-    protected function setUp()
+    protected function setUp(): void
```

Comment:
Why?

Reply:
I think my test complained, I have no (logged) other reason for this.

Reply:
Yeah:

`Return type declaration must be compatible with TestCase->setUp() : void`

Reply:
My concern here was about removing the docblock, sorry for the confusion